src/syntax/fujinet.syn

```diff
@@ -0,0 +1,72 @@
+#
+# FastBasic - Fast basic interpreter for the Atari 8-bit computers
+# Copyright (C) 2017-2022 Daniel Serpell
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+
+#
+# FujiNet syntax additions by Thomas Cherryhomes
+# <thom dot cherryhomes at gmail dot com>
+#
+
+STATEMENT:
+        "NOpen" EXPR emit { TOK_NUM_POKE, &$301 } \
```

Comment:
A lot of those could be factorized, making the parser smaller. I will work on this.